pyproject.toml

```diff
@@ -63,24 +63,24 @@ name = "Miscellaneous internal changes"
 showcontent = false
 
 [tool.poetry.dependencies]
-python = ">=3.7,<3.10"
+python = ">=3.7,<3.9"
```

Comment:
this is wrong though. I think `main` supports 3.9 🤔

Reply:
Thanks @m-vdb I'll make that change.